shared/src/iosMain/kotlin/co/nimblehq/kmm/template/di/module/PlatformModule.kt

```diff
@@ -0,0 +1,7 @@
+package co.nimblehq.kmm.template.di.module
+
+import org.koin.core.module.Module
+import org.koin.dsl.module
+
+actual fun platformModule(): Module = module {
+}
```

Comment:
Can we have //TODO: comment here too. 🙏🏻

Reply:
This is a valid comment 🙏

Reply:
@kaungkhantsoe did you miss this review?

Reply:
Ahh I thought I updated it. Sorry. Let me update it again.